Modules/Setup.stdlib.in

```diff
@@ -32,6 +32,7 @@
 @MODULE__ASYNCIO_TRUE@_asyncio _asynciomodule.c
 @MODULE__BISECT_TRUE@_bisect _bisectmodule.c
 @MODULE__CONTEXTVARS_TRUE@_contextvars _contextvarsmodule.c
+@MODULE__COPY_TRUE@_copy _copy.c
```

Comment:
If you include the build rule in ``Setup.stdlib.in`` then you have to update ``configure.ac`` and ``setup.py``, too.